pubsub/google/cloud/pubsub_v1/publisher/batch/thread.py

```diff
@@ -213,13 +220,13 @@ def monitor(self):
         This simply sleeps for ``self._settings.max_latency`` seconds,
         and then calls commit unless the batch has already been committed.
         """
-        # Note: This thread blocks; it is up to the calling code to call it
-        # in a separate thread.
-        #
+        # NOTE: This blocks; it is up to the calling code to call it
+        #       in a separate thread.
+
         # Sleep for however long we should be waiting.
         time.sleep(self._settings.max_latency)
```

Comment:
@lukesneeringer This could be a bug, e.g.

```python
>>> import time
>>> time.sleep(-1.0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: sleep length must be non-negative
```

Reply:
Happy to add `max(..., 0)`, but also happy to leave this be. :-)